build/gocommand.js

```diff
@@ -17,38 +17,120 @@
  */
 import child from 'child_process';
 import lodash from 'lodash';
+import Q from 'q';
```

Comment:
Style note: let's name this `q` and rename all capital Q to q in this file. This is how I see it used most often and it aligns with what is on the frontend. Small issue though